doc/modules/cross_validation.rst

```diff
@@ -261,25 +261,25 @@ two slightly unbalanced classes::
   [0 1 2 4 5 6 7] [3 8 9]
 
 
-Label k-fold
```

Comment:
Can you please organize it such that there is a "IID data" and a "Grouped data" section that explains the difference?
Maybe "Splitters for Independent Data (iid)" "Splitters for Grouped Data" and briefly explain each one?